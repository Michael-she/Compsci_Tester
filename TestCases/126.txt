l c 6 6
l c 4 6
l b 5 5
d b 5 6
#
6 6 r
4 6 r
5 6 r
5 7 r
5 5 r
5 6 r
