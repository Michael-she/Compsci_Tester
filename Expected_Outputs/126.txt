pygame 2.5.2 (SDL 2.28.3, Python 3.8.10)
Hello from the pygame community. https://www.pygame.org/contribute.html
   0  1  2  3  4  5  6  7  8  9
  +--+--+--+--+--+--+--+--+--+--+
9 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
8 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
7 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
6 |  |  |  |  |  |  | c|  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
5 |  |  |  |  |  | b| B|  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
4 |  |  |  |  |  |  | c|  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
3 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
2 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
1 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
0 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
   0  1  2  3  4  5  6  7  8  9
  +--+--+--+--+--+--+--+--+--+--+
9 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
8 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
7 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
6 |  |  |  |  |  |  |  | c|67|67|
  +--+--+--+--+--+--+--+--+--+--+
5 |  |  |  |  |  | b| B|  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
4 |  |  |  |  |  |  | c|  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
3 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
2 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
1 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
0 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
   0  1  2  3  4  5  6  7  8  9
  +--+--+--+--+--+--+--+--+--+--+
9 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
8 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
7 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
6 |  |  |  |  |  |  |  | c|67|67|
  +--+--+--+--+--+--+--+--+--+--+
5 |  |  |  |  |  | b| B|  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
4 |  |  |  |  |  |  |  | c|47|47|
  +--+--+--+--+--+--+--+--+--+--+
3 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
2 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
1 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
0 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
   0  1  2  3  4  5  6  7  8  9
  +--+--+--+--+--+--+--+--+--+--+
9 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
8 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
7 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
6 |  |  |  |  |  |  |  | c|67|67|
  +--+--+--+--+--+--+--+--+--+--+
5 |  |  |  |  |  | b|  | B|57|  |
  +--+--+--+--+--+--+--+--+--+--+
4 |  |  |  |  |  |  |  | c|47|47|
  +--+--+--+--+--+--+--+--+--+--+
3 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
2 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
1 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
0 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
   0  1  2  3  4  5  6  7  8  9
  +--+--+--+--+--+--+--+--+--+--+
9 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
8 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
7 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
6 |  |  |  |  |  |  |  | c|67|67|
  +--+--+--+--+--+--+--+--+--+--+
5 |  |  |  |  |  | b|  |  |  | B|
  +--+--+--+--+--+--+--+--+--+--+
4 |  |  |  |  |  |  |  | c|47|47|
  +--+--+--+--+--+--+--+--+--+--+
3 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
2 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
1 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
0 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
   0  1  2  3  4  5  6  7  8  9
  +--+--+--+--+--+--+--+--+--+--+
9 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
8 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
7 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
6 |  |  |  |  |  |  |  | c|67|67|
  +--+--+--+--+--+--+--+--+--+--+
5 |  |  |  |  |  |  | b|56|  | B|
  +--+--+--+--+--+--+--+--+--+--+
4 |  |  |  |  |  |  |  | c|47|47|
  +--+--+--+--+--+--+--+--+--+--+
3 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
2 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
1 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
0 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
   0  1  2  3  4  5  6  7  8  9
  +--+--+--+--+--+--+--+--+--+--+
9 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
8 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
7 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
6 |  |  |  |  |  |  |  | c|67|67|
  +--+--+--+--+--+--+--+--+--+--+
5 |  |  |  |  |  |  |  |  | b| B|
  +--+--+--+--+--+--+--+--+--+--+
4 |  |  |  |  |  |  |  | c|47|47|
  +--+--+--+--+--+--+--+--+--+--+
3 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
2 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
1 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
0 |  |  |  |  |  |  |  |  |  |  |
  +--+--+--+--+--+--+--+--+--+--+
Dark loses
